src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1218,6 +1218,20 @@ public void testEnvironmentVariableExpansion() throws Exception {
         assertTrue(r.hasChanges());
 
         build(project, Result.SUCCESS, "b.txt");
+
+        // Test branch expansion with environment contributors
+        FreeStyleProject project2 = createFreeStyleProject();
```

Comment:
Because injecting variables via other way is other test case. Every test should test single cases and not a scope of issues with similar name.